site/docs/tasks/control-plane-security.md

```diff
@@ -12,7 +12,83 @@ The `NetworkPolicy` is always created. However, your Kubernetes cluster must hav
 
 ## Mutual TLS
 
-If Istio is installed in the cluster then you can use [automatic mutual TLS](https://istio.io/latest/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls) to encrypt the traffic to the connectors.
+### Using Fybrik
+
+Fybrik can be configured to have the traffic between the manager and the connectors encrypted by using TLS. In addition, mutual TLS authentication is possible too.
```

Comment:
Fybrik can be configured to protect trafffic between the manager and connectors by using TLS.